lib/screens/MorePage.tsx

```diff
@@ -15,17 +15,21 @@ import { Ionicons } from "@expo/vector-icons";
 import { User } from "../models/user_class";
 import { useTheme } from "../components/ThemeProvider";
 import Accordion from "@gapur/react-native-accordion";
-import {useAuth0, Auth0Provider} from 'react-native-auth0';
+import { useDispatch, UseDispatch } from "react-redux";
+
 
 const user = User.getInstance();
 const { width, height } = Dimensions.get("window");
 
 export default function MorePage() {
 
   const { theme, isDarkmode, setIsDarkmode } = useTheme();
-  const {clearSession} = useAuth0();
 
   const toggleDarkmode = useTheme().toggleDarkmode;
+
+  //updated by karthik
```

Comment:
This is not a good comment to have in the source code, we can use git log to figure out who updated which line (if we need to figure it out).

Reply:
Sure, Dr. Holdener, the final version of the code will not reflect the issues mentioned above.